deepchem/splits/__init__.py

```diff
@@ -5,188 +5,291 @@
 from __future__ import division
 from __future__ import unicode_literals
 
-__author__ = "Bharath Ramsundar"
+__author__ = "Bharath Ramsundar, Aneesh Pappu "
 __copyright__ = "Copyright 2016, Stanford University"
 __license__ = "GPL"
 
 import os
 import numpy as np
+import pandas as pd
 from rdkit import Chem
 from deepchem.utils import ScaffoldGenerator
 from deepchem.utils.save import log
 from deepchem.datasets import Dataset
 from deepchem.featurizers.featurize import load_data
 
+
 def generate_scaffold(smiles, include_chirality=False):
-  """Compute the Bemis-Murcko scaffold for a SMILES string."""
-  mol = Chem.MolFromSmiles(smiles)
-  engine = ScaffoldGenerator(include_chirality=include_chirality)
-  scaffold = engine.get_scaffold(mol)
-  return scaffold
+    """Compute the Bemis-Murcko scaffold for a SMILES string."""
```

Comment:
A lot of the spacing in this file has been changed to 4-spaces. Could you revert everything to the 2-spacing format?